docs/sources/visualizations/canvas.md

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+  - /docs/grafana/latest/features/panels/canvas/
+  - /docs/grafana/latest/visualizations/canvas/
+description: Canvas visualization documentation
+keywords:
+  - grafana
+  - canvas
+  - panel
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+Introducing the Canvas panel, a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
```

Comment:
This page is not an announcement that it exists, it is just the docs.  Maybe start as:

"Canvas panel is..."